### PR TITLE
`var` interface change and docs

### LINKED ISSUE #1950: Make `var` return `Array<felt252>` instead of a single felt
### Which components does the task require to be changed? (think hard pls)

snforge

### Description

Sometimes the user will feel the need to use a longer variable (i.e. using a bytearray)
We should return serialized form of this in the array for the user to parse the value himself

The only thing that needs to be done, is that we currently return 1st value, whereas we should change the output type and return the whole array (describing the parsing behavior as analogous to `read_txt`/`read_json`)

We can also add a helper function akin to the `parse_txt` (being the helper for `read_txt`) which will parse this for the user, if the generic type implements `Serde`.

### PR DESCRIPTION
**Pull request by Arcticae**
Closes #1950

## Introduced changes

-

## Checklist

- [x] Linked relevant issue
- [x] Updated relevant documentation
- [x] Added relevant tests
- [x] Performed self-review of the code
- [x] Added changes to `CHANGELOG.md`

### CODE REVIEW

**Review by Draggu — APPROVED** [review]

File: docs/src/appendix/snforge-library/env/var.md [hunk 1]
```diff
@@ -1,21 +1,15 @@
 # `var`
 
-> `fn var(name: ByteArray) -> felt252`
+> `fn var(name: ByteArray) -> Array<felt252>`
 
-Read and parse felt252 or cairo short string (encoded as felt) value from an environment variable.
+Reads an environment variable, without parsing it.
 
-- `name` - name of an environment variable
+The serialized output is correlated with the inferred input type, as in the [file reading docs](../fs/read_txt.md#accepted-format).
```
Comment (piotmag769) [inline docs/src/appendix/snforge-library/env/var.md:7]:
```suggestion
The serialized output is correlated with the inferred input type, same as during [reading a file](../fs/read_txt.md#accepted-format).
```

File: crates/forge/tests/integration/env.rs [hunk 2]
```diff
@@ -47,6 +47,30 @@ fn read_felt252() {
...
+fn read_bytearray() {
+    let mut test = test_case!(indoc!(
+        r#"
+        use snforge_std::env::var;
+
+        #[test]
+        fn read_bytearray() {
+            let mut result = var("MY_ENV_VAR");
+            let result_bytearray = Serde::<ByteArray>::deserialize(ref result).unwrap();
+            assert(result_bytearray == "very long shorstring very very long very very long", 'failed reading env var');
```
Comment (piotmag769) [inline crates/forge/tests/integration/env.rs:60]:
```suggestion
            assert(result_bytearray == "very long string literal very very long very very long", 'failed reading env var');
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### Changed
 
 - `read_txt` and `read_json` now supports `ByteArray`
+- `var` now supports `ByteArray` with double quoting, and returns `Span<felt252>` instead of a single `felt252`
```
Comment (piotmag769) [inline CHANGELOG.md:15]: We should return an array

File: snforge_std/src/env/env_vars.cairo [hunk 4]
```diff
@@ -1,7 +1,9 @@
 use starknet::testing::cheatcode;
 use super::super::byte_array::byte_array_as_felt_array;
 
-fn var(name: ByteArray) -> felt252 {
-    let outputs = cheatcode::<'var'>(byte_array_as_felt_array(@name).span());
-    *outputs[0]
+/// Reads an environment variable, without parsing it
+/// `name` - name of an environment variable
+/// Returns the read span of felts
+fn var(name: ByteArray) -> Span<felt252> {
```
Comment (drknzz) [inline snforge_std/src/env/env_vars.cairo:7]: Here it's inconsistent with documentation (span / array)

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Forge
+
+#### Changed
+
+- `var` now supports `ByteArray` with double quoting, and returns `Span<felt252>` instead of a single `felt252`
```
Comment (piotmag769) [inline CHANGELOG.md:14]: Array?

**Review by piotmag769 — APPROVED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (Arcticae) [reply CHANGELOG.md:14]: forgor 💀

**Pull request merged**